app/controllers/organizations_controller.rb

```diff
@@ -216,8 +215,40 @@ def organization_params
     end
 
     if has_role 'organization_admin'
-      org_params.permit(:name, :export_type, :slug, :period_meta_key, :enable_workflows, :inherit_workflows_from_parents, :parent_id, :lms_authentication_source, :lms_authentication_id, :lms_authentication_key, :lms_info_slug, :lms_account_id, :home_page_redirect, :skip_lms_publish, :enable_shibboleth, :idp_sso_target_url, :idp_slo_target_url, :idp_entity_id, :idp_cert, :idp_cert_fingerprint, :idp_cert_fingerprint_algorithm, :authn_context, :enable_anonymous_actions, :track_meta_info_from_document, :disable_document_view,
```

Comment:
this has caused more than a few merge conflicts. i am guessing git will handle it better if its multiple lines.